examples/Demo/FluentUI.Demo.Client/Documentation/Components/Switch/FluentSwitch.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Switch
+route: /Switch
+---
+
+# Switch
+
+A **FluentSwitch** component represents a physical switch that allows someone to choose between two mutually exclusive options.  
+For example, "On/Off" and "Show/Hide". Choosing an option should produce an immediate result.
+
```

Comment:
Can you add the "Best practices" from the React page, adapted to our Blazor components?
https://react.fluentui.dev/?path=/docs/components-switch--docs

Reply:
ok, "Best practices" added in doc.

Reply:
We don't do that in other .md files: please don't use `###` but `**Layout**` and  `**Content**`.